src/kamel.ts

```diff
@@ -41,7 +41,7 @@ class KamelImpl implements Kamel {
 	}
 
 	async getPath(): Promise<string> {
-		return await baseKamelPath();
+		return (await baseKamelPath()).replace('\n', '');
```

Comment:
Do we know where the \n was slipping into the path?

Reply:
if I understood well, it is the "which" command that is used. the output is adding a \n to have the prompt going to a new line.

Reply:
wonderign if not better to replace in the baseKamelPath method then

Reply:
Yeah, that's kind of what I was thinking too -- might be better to move up stream a bit further?

Reply:
agree. done